src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/common.cljs

```diff
@@ -24,6 +19,15 @@
 (def call-running-statuses #{"NotStarted" "Starting" "Running"})
 (def call-failure-statuses #{"Failed" "Aborted"})
 
+(defn all-success? [submission]
+  (every? #(contains? wf-success-statuses (% "status")) (submission "workflows")))
```

Comment:
I think we still need to check if there are any workflows that failed to start.  And then for `any-failed?` do the opposite check.